front/components/TextInput/PasswordInput.spec.tsx

```diff
@@ -0,0 +1,51 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { Formik, Form } from 'formik';
+
+import TextInput from './index';
+
+test('PasswordInput', () => {
+  render(
+    <Formik
+      initialValues={{ search: '' }}
+      // eslint-disable-next-line no-console
+      onSubmit={(): void => console.log('submited')}
+    >
+      <Form>
+        <TextInput
+          disableHelperText
+          fullWidth
+          name="search"
+          type="search"
+          placeholder="search..."
+        />
+      </Form>
+    </Formik>,
+  );
+
+  const input = screen.getByPlaceholderText('search...');
+  expect(input).not.toHaveAttribute('aria-describedby', 'search-helper-text');
+});
+
+test('PasswordInput disableHelperText false', () => {
+  render(
+    <Formik
+      initialValues={{ search: '' }}
+      // eslint-disable-next-line no-console
+      onSubmit={(): void => console.log('submited')}
+    >
+      <Form>
+        <TextInput
+          disableHelperText={false}
+          fullWidth
+          name="search"
+          type="search"
+          placeholder="search..."
+        />
+      </Form>
+    </Formik>,
+  );
+
+  const input = screen.getByPlaceholderText('search...');
+  expect(input).toHaveAttribute('aria-describedby', 'search-helper-text');
+});
```

Comment:
It looks like this file has the wrong name as well as the test cases. Anyway, I don't think we need tests for the `TextInput` as there is no real logic in this component that doesn't come from Material UI.